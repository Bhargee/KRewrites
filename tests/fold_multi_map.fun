("foldl" (fun x -> fun y -> (x + y))) ("map" (fun z -> z - 1)) ("map" (fun d ->
d + 1))
