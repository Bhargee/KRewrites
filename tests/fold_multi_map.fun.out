<T>
    <k>
        ("foldl" (fun (#symExp(1) -> ((fun (x -> (fun (y -> (x + y))))) (
          (fun (z -> (z - 1))) #symExp(1)))))) ("map" (fun (d -> (d + 1))))
          
    </k>
    <store>
        .Map
    </store>
    <env>
        .Map
    </env>
</T>
